src/Uno.UWPSyncGenerator/AttributeGeneration/BindableAttributeDescription.cs

```diff
@@ -0,0 +1,21 @@
+﻿#nullable enable
+
+using Microsoft.CodeAnalysis;
+
+namespace Uno.UWPSyncGenerator.AttributeGeneration;
+
+internal class BindableAttributeDescription : AttributeDescriptionBase
+{
+	protected override string? TryGenerateAttributeParameters(AttributeData attributeData)
+	{
+		// This attribute doesn't have any constructor/named arguments.
+		// It was already handled by TryGenerateAttributeParametersCommon in AttributeDescriptionBase.
+		// We should never hit this code path, but if we did, just don't handle the attribute.
+		return null;
+	}
+
+	private protected override bool CanHandle(string fullyQualifiedAttributeName)
+	{
+		return fullyQualifiedAttributeName == "Windows" + ".UI.Xaml.Data.BindableAttribute";
```

Comment:
MUX is not needed here?

Reply:
There are no much types with BindableAttribute in WinUI it seems, the one I can see is https://learn.microsoft.com/en-us/windows/winui/api/microsoft.ui.xaml.controls.treeviewnode?view=winui-2.8, and is using `Windows.UI.Xaml.Data.BindableAttribute`. It wouldn't harm though to handle both here.